src/constructs/iam/policies/ses.test.ts

```diff
@@ -0,0 +1,48 @@
+import "@aws-cdk/assert/jest";
+import { SynthUtils } from "@aws-cdk/assert";
+import { attachPolicyToTestRole, simpleGuStackForTesting } from "../../../../test/utils";
+import { GuSESSenderPolicy } from "./ses";
+
+describe("GuSESSenderPolicy", () => {
+  it("should have a policy that reads from a parameter", () => {
+    const stack = simpleGuStackForTesting();
+    attachPolicyToTestRole(stack, new GuSESSenderPolicy(stack));
+
+    expect(stack).toHaveResource("AWS::IAM::Policy", {
+      PolicyDocument: {
+        Version: "2012-10-17",
+        Statement: [
+          {
+            Action: "ses:SendEmail",
+            Effect: "Allow",
+            Resource: {
+              "Fn::Join": [
+                "",
```

Comment:
This empty string is a cdk oddity isn't it? I was wondering if there was something to fix on our side, but it looks fine on our side

Reply:
Agreed, I don't think we should introduce a workaround for this.

Reply:
We might be able to use one of these functions https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_core.Arn.html.